File: apigateway/src/main/java/com/apigateway/dto/Room.java
package com.apigateway.dto;

public class Room {
	    
	public Long id;

	public String roomName;

	public String location;

	public int roomCapacity;
    
    public Long getId() {
        return id;
    }
    
    public void setId(Long id) {
        this.id = id;
    }
    
    public void setRoomName(String roomName) {
        this.roomName = roomName;
    }

    public String getRoomName() {
        return roomName;
    }

    public void setLocation(String location) {
        this.location = location;
    }

    public String getLocation() {
        return location;
    }
    
    public void setRoomCapacity(int roomCapacity) {
        this.roomCapacity = roomCapacity;
    }

    public int getRoomCapacity() {
        return roomCapacity;
    }
}

File: frontend/src/app/layout/rooms-page/room.ts
export class Room {
    roomName: string;
    location: string;
    roomCapacity: number;
    id: number;

    constructor(theName?: string, theLocation?: string, theCap?: number) {
        this.roomName = theName;
        this.location = theLocation;
        this.roomCapacity = theCap;
    }

}
